.github/workflows/welcome-bot.yml

```
@@ -1,3 +1,4 @@
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)